### PR TITLE
Document update for pagination

### PR DESCRIPTION
**Pull request by Aditi-sha**

### CODE REVIEW

File: 07-AzTS REST APIs/Scan Result APIs/Get Scan Results.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ POST https://<WebAPI-URL>/adhocscan/subscription/{subscriptionId}/ControlScanRes
 ```
 > _Note:_<br/>
 > _1. 'WebAPI-URL' varies per AzTS setup. You need to reach out to AzTS admin to get value for `<WebAPI-URL`> as mentioned [here](../README.md#setup-for-azts-admin-only)._<br/>
-> _2. 'api-version' is a required parameter. Current supported version: **1.0**_
+> _2. API will return 100 records of control scan result at a time. In order to get next set of records, you have to pass skipToken (available in API response) as part of next API request._<br/>
```
Comment (TarunKrShukla) [inline 07-AzTS REST APIs/Scan Result APIs/Get Scan Results.md:32]: As we have added details about require headers, request body can we details about the response bcz now its a complex object and not direct data.

File: 07-AzTS REST APIs/Scan Result APIs/Get Scan Results.md [hunk 2]
```diff
@@ -69,30 +71,48 @@ POST https://<WebAPI-URL>/adhocscan/subscription/{subscriptionId}/ControlScanRes
...
+        }
+    ],
+    "pageSize": 100,
+    "skipToken": "eyJQYWdlTnVtYmVyIjoyfQ==",
+    "totalPages": 5,
+    "totalRecords": 450,    
+}
 ```
 
+To get the next page of the results, send a request to the URL with skipToken. The skipToken contains next page number of control scan result. Continue sending requests with the skipToken until it no longer contains a skipToken in the returned results.
```
Comment (TarunKrShukla) [inline 07-AzTS REST APIs/Scan Result APIs/Get Scan Results.md:107]: We should mention somewhere skipToken will not be there if there are no more pages left,

File: 07-AzTS REST APIs/Scan Result APIs/Get Scan Results.md (hunk 1 above)
Comment (Aditi-sha) [reply 07-AzTS REST APIs/Scan Result APIs/Get Scan Results.md:32]: Added API response with details in proper tabular format.

File: 07-AzTS REST APIs/Scan Result APIs/Get Scan Results.md (hunk 2 above)
Comment (Aditi-sha) [reply 07-AzTS REST APIs/Scan Result APIs/Get Scan Results.md:107]: We have mentioned - "Continue sending requests with the skipToken until it no longer contains a skipToken in the returned results."

If this is not clear, we have added in response table as well.

**Review by TarunKrShukla — APPROVED** [review]
Looks good.

**Pull request merged**